cpp/tests/core/test_time.cpp

```diff
@@ -159,6 +159,7 @@ TEST( DateTimeEx, test_basic_functionality )
     ASSERT_EQ( ex2.microseconds(), 123456 );
     ASSERT_EQ( ex2.nanoseconds(),  123456789 );
 
+#ifdef __linux__
```

Comment:
i have code for this somewhere, let me find it

Reply:
its in the windows build branch, this should cover both

Reply:
im ok skipping this test if !linux